packages/vm/core/accounts/impl.go

```diff
@@ -178,6 +179,7 @@ func foundryCreateNew(ctx isc.Sandbox) dict.Dict {
 
 	ret := dict.New()
 	ret.Set(ParamFoundrySN, util.Uint32To4Bytes(sn))
+	ctx.Event(fmt.Sprintf("Foundry serial number = %d", sn))
```

Comment:
```suggestion
	ctx.Event(fmt.Sprintf("Foundry created, serial number = %d", sn))
```

Reply:
Shouldn't we use something more efficient? for the event payload? E.g.:

```suggestion
	ctx.Event(string(util.Uint32To4Bytes(sn)))
```

So that:
* it is more space efficient
* the client can decode the value more easily

Reply:
Hmmm but then we would need to include an "event type" code.

I think we should do this at some point. Let's merge this PR as is, and I'll add an issue to standardize the format of all events